forge-gui/res/cardsfolder/upcoming/mapping_the_maze.txt

```diff
@@ -0,0 +1,9 @@
+Name:Mapping the Maze
+ManaCost:1 U
+Types:Instant
+A:SP$ ChooseCard | ChoiceZone$ Hand,Graveyard | Choices$ Instant.YouOwn,Sorcery.YouOwn | ChoiceTitle$ Choose an instant or sorcery card in your hand or graveyard | SubAbility$ DBAnimate | SpellDescription$ Choose an instant or sorcery card in your hand or graveyard. 
+SVar:DBAnimate:DB$ Animate | Triggers$ SpellCastTrig | Defined$ ChosenCard | Incorporate$ U | Duration$ Perpetual | SubAbility$ DBReturn | SpellDescription$ That card perpetually incorporates {U} and gains "When you cast this spell, draw a card."
```

Comment:
this probably needs StackDescription of some sort?

Reply:
I don't think this is quite resolved

Reply:
Sorry resolved this too quickly. I think this way the stackdescription is fine ? 

![Uploading image.png…]()